.github/workflows/python.yml

```diff
@@ -2,8 +2,9 @@ name: Python
 
 on:
   push:
-    branches: [ master, python, experiments, stateful-c-lib ]
+    branches: [ master, python, experiments, iscgar/python-ci-fix ]
```

Comment:
This does not belong here, but only on your branch.

Reply:
Right. It's temporary, just to see the build run in my fork. I'll remove it once [the run](https://github.com/iscgar/solvespace/actions/runs/13475552864) completes.